libpod/oci_conmon_linux.go

```diff
@@ -956,6 +956,11 @@ func (r *ConmonOCIRuntime) createOCIContainer(ctr *Container, restoreOptions *Co
 
 	if ctr.config.PreserveFDs > 0 {
 		for fd := 3; fd < int(3+ctr.config.PreserveFDs); fd++ {
+			f := os.NewFile(uintptr(fd), fmt.Sprintf("fd-%d", fd))
```

Comment:
I think we need to do this in `cmd/podman` - we probably have at least this many open FDs of our own by this point (c/storage locks, etc)